packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -158,27 +158,35 @@ const main = async () => {
 
     managedRuntimeCommand('save-credentials')
         .description(`save API credentials for Managed Runtime`)
-        .requiredOption(
-            '-u, --user <email>',
-            'the e-mail address you used to register with Managed Runtime',
-            (val) => {
-                if (!validator.isEmail(val)) {
-                    throw new program.InvalidArgumentError(`"${val}" is not a valid email`)
-                } else {
-                    return val
+        .addOption(
+            new program.Option(
+                '-u, --user <email>',
+                'the e-mail address you used to register with Managed Runtime',
+                (val) => {
+                    if (!validator.isEmail(val)) {
+                        throw new program.InvalidArgumentError(`"${val}" is not a valid email`)
+                    } else {
+                        return val
+                    }
                 }
-            }
+            )
+                .env('MRT_USER')
```

Comment:
```suggestion
                .env('MRT_USER')
                .conflicts('credentialsFile')
```
Prevents the two from being used together.